views/show_space.templ

```diff
@@ -6,7 +6,12 @@ import (
 	"github.com/ugent-library/okay"
 )
 
-templ ShowSpace(c *ctx.Ctx, space *models.Space, folders []*models.Folder, q string, userSpaces []*models.Space, folder *models.Folder, errs *okay.Errors) {
+var sortOptions = []SelectOption{
+	{"default", "Expires first"},
```

Comment:
Why not just "expires-first"?

Reply:
Because I wanted to have a way that removes the sort query param from the URL if you go back to the default sort option ([code](https://github.com/ugent-library/deliver/blob/5102d7c1b340dc3ad6d1fe25bf339d15b3a195cc/models/pagination.go#L17)). This way it would work generically if we ever need sorting for files or another view.

It also corresponds to the default-clause in the switch statement that handles sorting for the query builder.

Reply:
But if "default" has one behaviour at one time, and another at another time (when you switch the default to something else), the expected behaviour will change for your end users. 

Better to keep this explicit?